src/main/resources/jenkins/branch/MultiBranchProject/configure-common.jelly

```diff
@@ -23,17 +23,11 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project" xmlns:this="this">
-  <p:config-disableBuild/>
-  <p:config-concurrentBuild />
 
   <st:include page="configure-basic.jelly" optional="true"/>
 
   <f:section title="${%Advanced Project Options}">
     <f:advanced>
-      <p:config-quietPeriod />
```

Comment:
`AbstractProject.quietPeriod` (anyway makes no sense for branch indexing that I can see…and as with `concurrentBuild`, if you wanted it on the branch project, we would need a separate system such as `BranchProperty`)